catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -144,6 +145,8 @@ public DeleteResponse delete(DeleteRequest deleteRequest)
                     deleteRequest.getAttributeValues()
                             .size());
 
+            deleteStorageRequest = new DeleteStorageRequestImpl(getDeleteMetacards(deleteRequest),
```

Comment:
One of my concerns during the recent refactor was the large number of side-effects. While I assume it's safe to create the `deleteStorageRequest` at this point, the inputs to the constructor are different than in the earlier invocation of the existing code. Are there any negative ramifications to that?